AUTHORS

```diff
@@ -1,24 +1,24 @@
 Ansible Container has been contribued to by the following authors:
 (This list is automatically generated - please file an issue for corrections)
 
-Chris Houseknecht <chouseknecht@ansible.com>
+Chris Houseknecht <house@redhat.com>
```

Comment:
See line 2 :) These changes won't stick. Ping me about how to set up the git pre-push hook.